Cargo.toml

```diff
@@ -46,6 +46,9 @@ toml-dep = { version = "0.5.8", package="toml", optional = true }
 aho-corasick = { version = "0.7.18", optional = true}
 rayon = { version = "1.5", optional = true}
 dbpnoise = { version = "0.1.2", optional = true}
+pathfinding = { version = "3.0.13", optional = true }
+mut_static = { version = "5.0.0", optional = true }
```

Comment:
This looks like a wacky library. What do you need it for? We already have both lazy_static and once_cell which should definitely be enough for your use case.